interfaces/cython/cantera/ck2cti.py

```diff
@@ -325,6 +326,9 @@ def to_cti(self, indent=0):
         if self.ID:
             kinstr = kinstr[:-1] + ",\n{0}id={1!r})".format(k_indent, self.ID)
 
+        if options:
+            kinstr = kinstr[:-1] + ",\n{0}options={1})".format(k_indent, optStr)
+
```

Comment:
Minor point, but I think it would be better to keep all the code for determining the `options=...` string together, so either this should be right after where `optStr` is set, or the rest of the code for setting `options` should be moved down here.